s4/clarity/artifact.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2016 Semaphore Solutions, Inc.
 # ---------------------------------------------------------------------------
-
+from s4.clarity._internal.factory import BatchFlags
```

Comment:
We still have to import it, just from .element instead of from .factory, which sucks, but it is a better place for the definition. I also explictly coded the default of none in.